projects/core/src/util/external-js-file-loader/external-js-file-loader.service.ts

```diff
@@ -23,30 +32,74 @@ export class ExternalJsFileLoader {
     callback?: EventListener,
     errorCallback?: EventListener
   ): void {
-    if (this.platformId && isPlatformServer(this.platformId)) {
-      if (errorCallback) {
-        errorCallback(new Event('error'));
-      }
+    this.addScript(
+      src,
+      params,
+      { type: 'text/javascript', async: true, defer: true },
+      callback,
+      errorCallback
+    );
+  }
+  /**
+   * Loads a javascript from an external URL.
+   * @param src URL for the script to be loaded
+   * @param params additional parameters to be attached to the given URL
+   * @param attributes the attributes of HTML script tag (exclude src)
+   * @param callback a function to be invoked after the script has been loaded
+   * @param errorCallback function to be invoked after error during script loading
+   * @param onSsr whether add script on SSR, by default false (CSR)
+   * @param inElement HTML body or head where script will be placed
+   */
+  public addScript(
+    src: string,
+    params?: Object,
+    attributes?: Object,
+    callback?: EventListener,
+    errorCallback?: EventListener,
+    onSsr = false,
```

Comment:
We discussed dropping `onSsr` in favor of guessing it from `callback` and `errorCallback` parameters being provided. 
So if there is callback or errorCallback, we assume that onSsr is true.